Sources/Identity/CustomerInfoManager.swift

```diff
@@ -189,6 +189,10 @@ class CustomerInfoManager {
     }
 
     func cachedCustomerInfo(appUserID: String) -> CustomerInfo? {
+        guard !self.systemInfo.dangerousSettings.uiPreviewMode else {
```

Comment:
Hmm I guess if we don't cache it, this would always be null and this might not be needed... But I guess I'm ok keeping it just in case.